lib/vendor/js-cookies.js

```diff
@@ -84,7 +84,23 @@ var Cookies = {
           break
       }
     }
-    document.cookie = [ encode(sKey), '=', encode(sValue), sExpires, sDomain ? '; domain=' + sDomain : '', sPath ? '; path=' + sPath : '', bSecure ? '; secure' : '' ].join('')
+
+    var secureAndSameSite = ''
+    if (sameSite && sameSite.toUpperCase() === 'NONE') {
+      // if SameSite is set to None, we need to add Secure
+      // otherwise setting cookie doesn't work in some browsers
+      secureAndSameSite = '; Secure; SameSite=' + sameSite
```

Comment:
We're overriding Secure here to true if customer deliberately set bsecure to false. IMHO, we should either document this, or honor customer config secure = false if they set it.

Reply:
I'll mention this in document